pkg/testing/ess/create_deployment_csp_configuration.yaml

```diff
@@ -14,6 +14,6 @@ aws:
   elasticsearch_deployment_template_id: "aws-storage-optimized"
   kibana_instance_configuration_id: "aws.kibana.c5d"
 docker:
-  integration_server_image: "docker.elastic.co/cloud-release/elastic-agent-cloud:9.1.0-6afab230-SNAPSHOT"
-  elasticsearch_image: "docker.elastic.co/cloud-release/elasticsearch-cloud-ess:9.1.0-6afab230-SNAPSHOT"
-  kibana_image: "docker.elastic.co/cloud-release/kibana-cloud:9.1.0-6afab230-SNAPSHOT"
\ No newline at end of file
+  integration_server_image: "docker.elastic.co/cloud-release/elastic-agent-cloud:${OVERRIDE_STACK_VERSION}"
+  elasticsearch_image: "docker.elastic.co/cloud-release/elasticsearch-cloud-ess:${OVERRIDE_STACK_VERSION}"
+  kibana_image: "docker.elastic.co/cloud-release/kibana-cloud:${OVERRIDE_STACK_VERSION}"
```

Comment:
I'm not sure about the changes, IIUC, https://github.com/elastic/elastic-agent/pull/7098/files#diff-38acbfb3e60b4cc22598e92b85dda23797dec02f252687606910b58687a89d60R65-R79 is the one actually removing the pinned version.

What's the reason `stack_version` is not override with the value of `OVERRIDE_STACK_VERSION`?